backend/constant/errs.go

```diff
@@ -126,10 +126,11 @@ var (
 
 // container
 var (
-	ErrInUsed       = "ErrInUsed"
-	ErrObjectInUsed = "ErrObjectInUsed"
-	ErrPortRules    = "ErrPortRules"
-	ErrPgImagePull  = "ErrPgImagePull"
+	ErrInUsed            = "ErrInUsed"
+	ErrObjectInUsed      = "ErrObjectInUsed"
+	ErrObjectBeDependent = "ErrObjectBeDependent"
+	ErrPortRules         = "ErrPortRules"
+	ErrPgImagePull       = "ErrPgImagePull"
 )
 
 // runtime
```

Comment:
The code you've provided is primarily defining constants for error messages in Go using the `var` keyword. There are no specific errors listed in your code snippet that indicate a problem or require further optimization. However, here are some general guidelines for maintaining clean and efficient constant definitions:

1. **Consistent Naming**: Ensure all constant names follow consistent naming conventions (e.g., use underscores to separate words).

2. **Avoid Ambiguity**: Clearly differentiate between "in_use" and "be_dependent". The name `ErrObjectBeDependent` could be clearer.

3. **Error Messages**: While clear error messages are important and should communicate effectively, they can sometimes become long and cumbersome if excessively detailed.

4. **Reusability**: Consider whether these error strings will need to change frequently or across different versions of your application. If so, it might be beneficial to store them in environment variables or configuration files instead of hardcoding them into your source code.

5. **Documentation**: Include comments explaining each constant and what each represents. This is particularly useful for other developers who may work on this code in the future.

Overall, your code structure looks neat and maintainable for the purpose of defining error messages in a struct.